packages/swayze/lib/src/core/controller/cells/cells_controller.dart

```diff
@@ -238,9 +241,18 @@ class SwayzeCellsController<CellDataType extends SwayzeCellData>
     // finds the a suitable coordinate.
     do {
       newCoordinate += _moveVectors[direction]!;
+
+      final dx = maxElasticColumns != null
+          ? min(newCoordinate.dx, maxElasticColumns - 1)
+          : newCoordinate.dx;
+
+      final dy = maxElasticRows != null
+          ? min(newCoordinate.dy, maxElasticRows - 1)
+          : newCoordinate.dy;
```

Comment:
this prevents the user from being able to select cells that exceed the max amount of elastic columns or rows